### DISCUSSION TITLE
In some tables CSV export fails [Jun 30]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#6027] [Discord Threads]
> 🙋‍♂️ **gab1to**  
## Hello guys, I've been trying to solve this but I still have no solution for it. In previous versions of NocoDB I could export the data to a csv but now, after the update I get this error. In only happens in some tables and I can't figure out what am I doing wrong. Any help will be appreciated.  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/1113519610810089562/nocodb.PNG)

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### Can u pls DM any server/browser logs  
#  
> 💁 **o1lab**  
#### if there is a date or datetime field in that grid view - can u disable using `fields` menu and try downloading it again  
#  
> 👨‍💻 **pranavxc**  
#### Can you add a sort option and try out exporting  
#  
> 🙋‍♂️ **gab1to**  
#### there's a date field that is empty, I disabled all fields and left only one field and sort the list to only one result and still got the same error  
#  
> 👨‍💻 **pranavxc**  
#### Does the table have primary key ?  
#  
> 👨‍💻 **pranavxc**  
#### auto incremented primary key or `created_at` field  
#  
> 🙋‍♂️ **gab1to**  
#### well, this is a View  
#  
> 👨‍💻 **pranavxc**  
#### but the actual table have those fields or not ?  
#  
> 👨‍💻 **pranavxc**  
#### the generated select query doesn't have any `order by` clause and which causing the error  
#  
> 🙋‍♂️ **gab1to**  
#### ok I'll look into it  
#  
> 🙋‍♂️ **gab1to**  
#### we added the order by to the view, still got the same error  
#  
> 🙋‍♂️ **gab1to**  
#### I'm happy to report that the solution was Sorting the data inside NocoDB 😆  
#  
> 🙋‍♂️ **gab1to**  
#### thanks a lot guys  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1113519610701041725)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
